addons/web/static/src/webclient/burger_menu/burger_menu.scss

```diff
@@ -4,7 +4,7 @@
 
 .o_burger_menu, .o_app_menu_sidebar {
     width: Min(360px, 80%);
-    z-index: $zindex-tooltip + 10;
+    z-index: $zindex-modal;
```

Comment:
This downgrades the z-index from 1090 to 1055.
Many main components are around 1080 so I don't think this is really what we want.
Maybe best to make the custom dropdown/bottom sheet there with higher z-index?